docs/examples/crawler_login.mdx

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Also wondering whether this is more of an example or a guide?

Reply:
I think a guide would work better. Over time, we should probably add more examples to this page (describe different approaches) and explain things in a bit more detail.